gradle/dependencies.gradle

```diff
@@ -34,13 +34,13 @@ versions += [
   junit: "4.12",
   lz4: "1.3.0",
   metrics: "2.2.0",
-  powermock: "1.6.3",
+  powermock: "1.6.4",
   reflections: "0.9.10",
   rocksDB: "3.10.1",
```

Comment:
@ijuma I used an older version of RocksDB since it was stable at that time, and I think we should upgrade it some time.